src/args.rs

```diff
@@ -47,6 +47,12 @@ fn app() -> Command<'static> {
                 .value_name("path")
                 .help("Specify an url path prefix"),
         )
+        .arg(
+            Arg::new("basic-auth")
+                .short('B')
+                .long("basic-auth")
+                .help("Use HTTP basic auth instead of digest auth"),
+        )
```

Comment:
It's better to provicde choices than switcher
```suggestion
        .arg(
            Arg::new("auth-method")
                .long("auth-method")
                .help("Choose auth method")
                .possible_values(["basic", "digest"])
                .default_value("digest")
                .value_name("value"),
        )
```